src/helpers/StakedCeloProvider.ts

```diff
@@ -0,0 +1,47 @@
+import { JsonRpcProvider } from "@ethersproject/providers"
+import BigNumber from "bignumber.js"
+import { ethers } from "ethers"
+
+const STAKED_CELO_MANAGER_ADDRESS = "0x0239b96D10a434a56CC9E09383077A0490cF9398"
+const STAKED_CELO_ERC20_ADDRESS = "0xC668583dcbDc9ae6FA3CE46462758188adfdfC24"
+
+const STAKED_CELO_ERC20_ABI = [
```

Comment:
there is already a ERC20_ABI in src/constants/abis.ts
to keep the code consistent i would this same for the addresses in src/contract-addresses.ts

Reply:
Good point! Made the changes!